chat-plugins/info.js

```diff
@@ -36,6 +36,7 @@ exports.commands = {
 		}
 
 		let buf = Chat.html`<strong class="username"><small style="display:none">${targetUser.group}</small>${targetUser.name}</strong> `;
+		if (targetUser.autoconfirmed) buf += ` <small style="color:gray">(ac)</small>`;
```

Comment:
I don't think this would leak "alts", but it could potentially leak "previous names", and those are already kind of public

That's my thought process, at least